### DISCUSSION TITLE
Windows Package Manager 1.7.11132

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#4403] [Releases]
This release is the fourth stable release of Windows Package Manager 1.7 for Windows 10 (1809+), and Windows 11.
Experimental features have been disabled in this release.

This release addresses various bugs and adds arm64 support for the Microsoft.WinGet.Client powershell module.

## What's Changed
* Add support for optional limitation set in configuration remote server and processors by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4349
* Improve MSStore installation success rate by trying Restart or Cancel when applicable by @yao-msft in https://github.com/microsoft/winget-cli/pull/4356
* Dynamic runtime configuration support by @JohnMcPMS and @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4355
* Make DLL directory work in packaged cases as well by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4389
* Enable Microsoft.WinGet.Client arm64 support by @msftrubengu in https://github.com/microsoft/winget-cli/pull/4392

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.7.10861...v1.7.11132

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.7.11132'>Windows Package Manager 1.7.11132</a>.</em>

**Comment by Karl-WE** [comment]
Is the Windows SDK for Windows 11 (10.0.22000.194 something we need to look into getting updated as a prerequisite of winget?

I would also like to propose, the experimental feature Auto elevation to be included into the brief release notes for larger adoption / testing.

**Comment by tonnesen-ncqa** [comment]
I feel a little crazy, but the files that are downloading from this release don't match the files listed.

Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle is downloading as:
Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.txt.msixbundle

The license file is downloading a zip file, the zip file is downloading something else.
